src/styles/components/buttonStyles.ts

```diff
@@ -18,6 +18,17 @@ const buttonStyles = {
       _hover: { bg: "brand.secondary", opacity: "0.60" },
       _active: { bg: "brand.secondary", opacity: "0.60" },
     },
+    checkoutBtn: {
```

Comment:
Pls make sure that you pull the origin/main and merge your branch from the local/main first, before you start working.

Reply:
OK, I have pulled the latest origin/main. Thank you.